torch/fx/experimental/symbolic_shapes.py

```diff
@@ -41,9 +71,23 @@ def handle_symbolic_op(func, args, kwargs):
     if func == torch.ops.aten.is_contiguous.default:
         return True
     # TODO: hack, we don't currently support symbolic strides properly
+    # NB: this results in goop in the trace, it will be fixed when we have
+    # proper support
     if func == torch.ops.aten.stride.default:
         return create_contiguous(args[0].shape)
 
+def _handle_sym_dispatch(func, args, kwargs):
+    global SYM_FUNCTION_MODE
+    mode = SYM_FUNCTION_MODE
+    assert mode
+    SYM_FUNCTION_MODE = mode.inner
```

Comment:
hmm why do we get `mode.inner` here ?

Reply:
It's a mode stack, so you remove the top mode from the stack but have the inner mode set